great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -1458,8 +1474,7 @@ def run_checkpoint(
         expectation_suite_ge_cloud_id: Optional[str] = None,
         **kwargs,
     ) -> CheckpointResult:
-        """
-        Validate against a pre-defined Checkpoint. (Experimental)
+        """Validate against using an existing Checkpoint.
```

Comment:
"Validate against using"? I think we want either "against" or "using".

Reply:
fixed, thanks!